include/seqan3/alignment/configuration/all.hpp

```diff
@@ -26,19 +42,3 @@
 #include <seqan3/alignment/configuration/align_config_scoring_scheme.hpp>
 #include <seqan3/alignment/configuration/align_config_vectorised.hpp>
 #include <seqan3/alignment/configuration/detail.hpp>
```

Comment:
oh spotted a detail here? I guess you just added inlcudes but do not touch existing ones. fine for now in the scope of this PR! This can be a follow up.

Reply:
Mhm, interessting. The issue is, that I didn't plan for `detail.hpp` only for `detail` folders....
Nice spot!
Will fix it now otherwise I forget.